planner/core/memtable_predicate_extractor.go

```diff
@@ -425,14 +425,14 @@ func (helper extractHelper) extractTimeRange(
 					endTime = mathutil.MinInt64(endTime, timestamp)
 				}
 			case ast.GT:
-				startTime = mathutil.MaxInt64(startTime, timestamp+1)
+				startTime = mathutil.MaxInt64(startTime, timestamp+int64(time.Millisecond))
```

Comment:
How about `timestamp + int64(time.Nanosecond)`

Reply:
Use `Millisecond ` here is because of The duration precision of `Log Search` is `Millisecond`. Use `Nanosecond` here will useless.

Actually currently, the process here is not absolutely correct. But this is an another issue, I will add a todo here.